guides/source/active_record_validations.md

```diff
@@ -991,7 +1000,8 @@ class Invoice < ActiveRecord::Base
 end
 ```
 
-By default such validations will run every time you call `valid?`. It is also
+By default such validations will run every time `valid?` is called, whether
+it is called by your code or by Rails before the object is saved. It is also
```

Comment:
Hmm... by this point, it feels like we're over-selling the "Rails will call `valid?` while saving" bit -- which (AFAIK) isn't even part of any published contract. (We promise that AR will validate on save, and that `valid?` will validate... but not necessarily that we'll use that particular method internally.)